setup.env

```diff
@@ -1,38 +1,70 @@
-### Change the LSST_SDK_INSTALL and OSPL_HOME to the actual locations and uncomment
+### Script to setup SAL environment. Default config assumes ts_opensplice is
+### available in the same directory as ts_sal.
 ###
-### export LSST_SDK_INSTALL=/home/saluser
-### export OSPL_HOME=$LSST_SDK_INSTALL/OpenSpliceDDS/V6.9/HDE/x86_64.linux
+### To get required projects from git, do:
 ###
-### export PYTHON_BUILD_VERSION=3.6m
-### export PYTHON_BUILD_LOCATION=/usr/local
+### mkdir lsst && cd lsst  # or some other directory
+### git clone https://github.com/lsst-ts/ts_opensplice
+### git clone https://github.com/lsst-ts/ts_sal
+### source ts_sal/setup.env
```

Comment:
ts_sal also wants ts_idl to be installed. And, of course ADLink Vortex OpenSplice.